components/FilterCategory/FilterCategory.Styled.js

```diff
@@ -0,0 +1,6 @@
+import styled from "styled-components";
+
+export const StyledDropdown = styled.select`
+  margin-left: 5px;
+  padding: 5px 0 5px 0;
```

Comment:
When the padding values for top and bottom are the same, and left and right paddings are 0, you can simplify the notation by specifying only two values.

```suggestion
  padding: 5px 0;
```